### DISCUSSION TITLE
command difference.

### DISCUSSION THREAD

**Discussion by sriniwaasg** [learndevopsonline/learndevopsonline#1282] [Q&A] [answered]
Hi Raghu,
   what is the difference between below two commands and why comman.sh is not found in second command.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129134722/5622fbe3-a34b-4f85-8e95-25aa159a20ab)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129134722/0eff4789-baff-42b8-8805-abb370a3965c)

**Comment by Naresh-vk-243** [comment, marked as answer]
hi @sriniwaasg, as per my understanding the sudo command will have access to user.sh only and the source file is different and we need to go to that common.sh file to access that function, so the sudo command wont have access for that common.sh file, so it cant find the common.sh file, so it will throw an error as it is not in the roboshop-shell directory, hope you understand .....
